src/task_processor.hpp

```diff
@@ -53,9 +53,13 @@ class TaskProcessor {
         task_(task),
         threads_(executor_queue.size()) {
     // Make as many threads as there are executors.
+    threads_rejoined_ = false;
```

Comment:
This line seems a little out of place relative to the comment above.

Reply:
This variable was optimized out.